vignettes/pr_review_guidance.Rmd

```diff
@@ -181,65 +177,59 @@ for (pkg in base_recommended_pkgs) {
 assign(".lib.loc", ".library", envir = environment(.libPaths))
 
 r_version <- getRversion()
-if (grepl("^3.6", r_version)) {
-  options(repos = "https://cran.microsoft.com/snapshot/2020-02-29")
-} else if (grepl("^4.0", r_version)) {
-  options(repos = "https://cran.microsoft.com/snapshot/2021-03-31")
+if ([grepl]("^4.0", r_version)) {
+  [options](repos = "https://cran.microsoft.com/snapshot/2021-03-31")
+} else if ([grepl]("^4.1", r_version)) {
+  [options](repos = "https://cran.microsoft.com/snapshot/2022-03-10")
+} else if ([grepl]("release", r_version)) {
+  [options](repos = "https://cran.microsoft.com/snapshot/2022-06-23")
 } else {
-  options(repos = "https://cran.rstudio.com")
+  [options](https://rdrr.io/r/base/options.html)(repos = "https://cran.rstudio.com")
 }
```

Comment:
Please remove the squared brackets around `grepl` and `options`. Otherwise the code fails if copied into the console.